src/ucp/core/ucp_mm.c

```diff
@@ -99,10 +100,12 @@ ucs_status_t ucp_mem_rereg_mds(ucp_context_h context, ucp_md_map_t reg_md_map,
             status = uct_md_mem_reg(context->tl_mds[md_index].md, address,
                                     length, uct_flags, &uct_memh[memh_index]);
             if (status != UCS_OK) {
-                ucs_error("failed to register address %p length %zu on md[%d]=%s: %s",
-                          address, length, md_index,
-                          context->tl_mds[md_index].rsc.md_name,
-                          ucs_status_string(status));
+                level = (uct_flags & UCT_MD_MEM_FLAG_HIDE_ERRORS) ?
+                        UCS_LOG_LEVEL_DEBUG : UCS_LOG_LEVEL_ERROR;
```

Comment:
Does this log message belong at debug level if errors are hidden? It seems more high volume (trace) level since presumably a failure to register the address when errors are hidden is just going to trigger some other appropriate action that will be logged.

Reply:
@gmegan, is this PR good to go?